src/altimate.ts

```diff
@@ -110,6 +110,23 @@ export class AltimateRequest {
     return !!this.getConfig();
   }
 
+  handlePreviewFeatures(): boolean {
+    if (this.enabled()) {
+      return true;
+    }
+    window
+      .showInformationMessage(
+        `You need Altimate AI API Key to use this feature. Get your API Key for free by signing up`,
+        "Sign Up for API Key",
+      )
+      .then((answer) => {
```

Comment:
use await and async and use a enum or a constant for the option, there are examples.